src/officehours_ui/templates/base.html

```diff
@@ -3,112 +3,140 @@
 {% get_flatpages as flatpages %}
 <!doctype html>
 <html lang="en">
-  <head>
-  {% block head %}
-	<link rel="shortcut icon" type="image/png" href="{% static 'images/favicon.png' %}"/>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
-  <link href="{% static 'fontawesomefree/css/all.min.css' %}" rel="stylesheet" type="text/css">
-  <link rel="stylesheet" type="text/css" href="{% static 'css/main.css' %}">
-	<link rel="stylesheet" mo type="text/css" href="//fonts.googleapis.com/css?family=Roboto:400"/>
-	<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:400i"/>
-	<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:700"/>
-	<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:700i"/>
-  {% block styles %}{% endblock %}
-  <title>Remote Office Hours Queue - {% block title %}{% endblock %}</title>
-  {{ spa_globals|json_script:"spa_globals" }}
-  {% endblock %}
-  </head>
+ <head>
+ {% block head %}
+ <link rel="shortcut icon" type="image/png" href="{% static 'images/favicon.png' %}"/>
+ <meta charset="utf-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+ <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+ <link href="{% static 'fontawesomefree/css/all.min.css' %}" rel="stylesheet" type="text/css">
+ <link rel="stylesheet" type="text/css" href="{% static 'css/main.css' %}">
+ <link rel="stylesheet" mo type="text/css" href="//fonts.googleapis.com/css?family=Roboto:400"/>
+ <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:400i"/>
+ <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:700"/>
+ <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:700i"/>
+ {% block styles %}{% endblock %}
+ <title>Remote Office Hours Queue - {% block title %}{% endblock %}</title>
+ {{ spa_globals|json_script:"spa_globals" }}
+ {% endblock %}
+ </head>
```

Comment:
these are the formatting changes you made, maybe try copying and pasting from the old version and see if the diff changes

Reply:
Ok I think it should be good now let me know if I missed something!

Reply:
I don't know if it's exactly how it was before but at least it hopefully looks more intuitive than whatever it was when I originally pushed it